### PR TITLE
Cut release v1.0.0

### PR DESCRIPTION
**Pull request by beorn7**
@krasi-georgiev please let me know if you see any blockers. We can also incorporate #603 if that makes sense.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+## 1.0.0 / 2019-06-15
+
+_This release removes all previously deprecated features, resulting in the breaking changes listed below._
```
Comment (brian-brazil) [inline CHANGELOG.md]: I'd add a mention that you're now following semver (except for API).

**Comment by krasi-georgiev** [comment]
> We can also incorporate #603 if that makes sense.

yep I just merged it.

**Comment by beorn7** [comment]
Totally missed #599. Will add.

**Comment by beorn7** [comment]
All comments addressed.

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**